src/Sulu/Bundle/SecurityBundle/Behat/SecurityContext.php

```diff
@@ -136,6 +160,31 @@ public function theNotEnabledUserExistsWithPassword($username, $password)
     public function iAmLoggedInAsAnAdministrator()
     {
         $this->theUserExistsWithPassword('admin', 'admin');
+
+        $this->visitPath('/admin');
+        $page = $this->getSession()->getPage();
+        $this->waitForSelector('#username');
+        $this->fillSelector('#username', 'admin');
+        $this->fillSelector('#password', 'admin');
+        $loginButton = $page->findById('login-button');
+
+        if (!$loginButton) {
+            throw new \InvalidArgumentException(
+                'Could not find submit button on login page'
+            );
+        }
+
+        $loginButton->click();
+        $this->getSession()->wait(5000, "document.querySelector('.navigation')");
```

Comment:
Why do we add this code (again?)? You can just add $locale = null to this method and call it in `iAmLogg..AndLocale` below

```
public function iAmLoggedInAsAnAdmistrator($locale = null)
```

or?